dev/idm-am-common-user/idm/conf/endpoint-usernotifications.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
You probably don't need this either. I only added it for backwards compatibility with 6.0 end-user clients which want to be able to read notifications using the same API as 6.0.